homeassistant/components/device_automation/__init__.py

```diff
@@ -59,17 +62,33 @@
     for entity in entities:
         domains.add(split_entity_id(entity.entity_id)[0])
 
-    device_triggers = await asyncio.gather(
+    device_automations = await asyncio.gather(
         *(
-            _async_get_device_automation_triggers(hass, domain, device_id)
+            _async_get_device_automations(hass, domain, fname, device_id)
             for domain in domains
         )
     )
-    for device_trigger in device_triggers:
-        if device_trigger is not None:
-            triggers.extend(device_trigger)
+    for device_automation in device_automations:
+        if device_automation is not None:
+            automations.extend(device_automation)
+
+    return automations
 
-    return triggers
+
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "device_automation/condition/list",
+        vol.Required("device_id"): str,
+    }
+)
+async def websocket_device_automation_list_conditions(hass, connection, msg):
+    """Handle request for device conditions."""
+    device_id = msg["device_id"]
+    conditions = await async_get_device_automations(
+        hass, "async_get_conditions", device_id
+    )
+    connection.send_result(msg["id"], {"conditions": conditions})
```

Comment:
Why would we add this to a dictionary? We should just send the list.
```suggestion
    connection.send_result(msg["id"], conditions)
```

We should do the same for triggers (which I thought we already did)

Reply:
Because someone said it's better to add to a dict key in case we want to extend in the future. We actually did the reverse change for triggers, it started as just a list IIRC.